scripts/create-root-user.py

```diff
@@ -26,4 +28,14 @@
         print 'Passwords don\'t match. Try again.'
         print
 
-model.add_user(username, '', password, user_id, is_verified=True)
+try:
```

Comment:
I think it would be better if you explicitly tested to see whether the user exists, and then either add or update accordingly, rather than relying on a generic Exception being caught (failure could have been to multiple reasons).